### PR TITLE
FIX HTML showing in inline preview

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-elemental-fileblock/issues/39

### CODE REVIEW

File: src/Block/FileBlock.php
Comment (ChloeSartorelli) [inline src/Block/FileBlock.php:1]: Please mark the `getSummaryThumbnail()` method as deprecated - we should remove it in CMS 6.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Overall looks good. Just one small change.

File: src/Block/FileBlock.php
Comment (sabina-talipova) [reply src/Block/FileBlock.php:1]: I've added two new PR for this.
- https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/43
- https://github.com/silverstripe/developer-docs/pull/300

File: src/Block/FileBlock.php [hunk 1]
```diff
@@ -60,17 +60,26 @@ public function getSummary()
...
         }
+
         return $blockSchema;
     }
 
     /**
      * Return a thumbnail of the file, if it's an image. Used in GridField preview summaries.
      *
      * @return DBHTMLText
+     * @deprecated 3.1 Will be removed without equivalent functionality to replace it
```
Comment (sabina-talipova) [inline src/Block/FileBlock.php]: I marked method as deprecated here.

Comment (ChloeSartorelli) [reply src/Block/FileBlock.php]:
```suggestion
```
Please remove that - I agree with your assessment that it shouldn't be marked as deprecated until the next minor release which is covered by https://github.com/silverstripe/silverstripe-elemental-fileblock/pull/43

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by sabina-talipova** [comment]
Updated

**Review by GuySartorelli — APPROVED** [review]
MOG

**Pull request merged**